vmware_blockchain/README.md

```diff
@@ -0,0 +1,34 @@
+## VMware Blockchain Setup
+
+VMware Blockchain is an enterprise-grade blockchain platform that enables multi-party workflows. It uses advanced techniques, such as byzantine fault-tolerant state machine replication, authenticated data structures, and integration with smart contract execution engines, to enable customers to build and run decentralized multi-party applications.
+
+### Step 1. Install the Wavefront Proxy
+
+If you do not have a [Wavefront proxy](https://docs.wavefront.com/proxies.html) installed on your network, install a proxy. Navigate to Browse > Proxies, click Add new proxy, and follow the instructions.
```

Comment:
**Browse > Proxies** and **Add new proxy** should be bold, because they are UI elements.